dsl/src/nativeMain/kotlin/com/freeletics/flowredux/dsl/util/AtomicCounter.kt

```diff
@@ -0,0 +1,17 @@
+package com.freeletics.flowredux.dsl.util
+
+import kotlin.native.concurrent.AtomicInt
+
+public actual class AtomicCounter actual constructor(initialValue: Int) {
+    private val atom = AtomicInt(initialValue)
+
+    public actual fun get(): Int = atom.value
+
+    public actual fun set(newValue: Int) {
```

Comment:
Nitpicking, but I also think `set` also doesn't need to be public API as we dont need it right now (just get and increment, decrement). But not a blocker, we can clean that up later imho
I